src/popup/pages/SecureLoginSettings.vue

```diff
@@ -30,15 +30,15 @@
             </div>
 
             <RadioButton
-              v-for="(ms, minutes) in AUTHENTICATION_TIMEOUTS"
-              :key="`timeout-${minutes}`"
+              v-for="(ms) in AUTHENTICATION_TIMEOUTS"
+              :key="`timeout-${ms}`"
               :value="secureLoginTimeout === ms"
               :class="{ active: secureLoginTimeout === ms }"
               class="timeout"
               has-label-effect
               @input="setSecureLoginTimeout(ms)"
             >
-              <div class="row" v-text="$t('pages.secureLogin.authenticationTimeout', Number(minutes))" />
+              <div class="row" v-text="$t('pages.secureLogin.authenticationTimeout', msToMinutes(ms))" />
```

Comment:
I suggest to use `label` property instead.

Reply:
`label` is added inside the `RadioButton.vue` component. I guess `span` tag is more appropriate.